src/main/java/Deadline.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+
+/**
+ * Represents a Deadline which is a subclass of Task
+ * Includes a dueDate attribute. Overrides toString() from Task
+ */
+public class Deadline extends Task {
+    private String dueDate;
+    public Deadline(String name, String time) {super(name); this.dueDate = time;}
+
+    /**
+     * @override
+     * @return String of Deadline task, eg [D][X] Deadline (by:XX)
+     */
```

Comment:
@Override should not be in the javadoc. Missing description for the method in the javadoc.
Recommendation
```
/**
* Prints the String representation for the Deadline task.
* 
* @return String of Deadline task, eg [D][X] Deadline (by:XX)
* /
@Override
public String toString() {...}
```